crates/hyperdrive-wrappers/build.rs

```diff
@@ -58,6 +58,12 @@ const TARGETS: &[&str] = &[
 
 fn get_artifacts(artifacts_path: &Path) -> Result<Vec<(String, String)>> {
     let mut artifacts = Vec::new();
+    if !artifacts_path.exists() {
```

Comment:
We could just build it here with mkdirp. It’s easy enough that I think it’s worth doing

Reply:
oh ok cool, it just needs to be an empty dir? or do we need to populate it with something?